cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md

```diff
@@ -26,6 +26,9 @@ runtime-spec-compatible `dest/config.json`.
 **--type**
   Type of the file to unpack. If unset, oci-create-runtime-bundle will try to auto-detect the type. One of "imageLayout,image"
 
+**--same-owner**
+  Preserve the owner and group of the layer entries when unpacking the image as a non-root user. For root this is the default behaviour. (default false)
```

Comment:
I'd avoid “(default false)”, since that's not true for root.  How about echoing tar(1) and doing something like:

> Preserve the owner and group of the layer entries when unpacking the image (default for superuser).

The explicit scoping there makes it fairly clear that it is not the default for ordinary users, but I'm also ok making that explict with something like:

> Preserve the owner and group of the layer entries when unpacking the image (default for superuser, but not for ordinary users).